src/components/ArticlesPage/ArticleCard.jsx

```diff
@@ -0,0 +1,22 @@
+import "./ArticleCard.css";
+
+function ArticleCard({ article }) {
+    const {created_at} = article
+    const postedDate = created_at.slice(8, 10) + "/" + created_at.slice(5, 7) + "/" + created_at.slice(0, 4)
+    const postedTime = created_at.slice(11, 16)
+
+    return (
+        <article>
+            <h3>{article.title}</h3>
+            <p>Topic: {article.topic}</p>
+            <p>Votes: {article.votes}</p>
+            <p>Posted by: {article.author}</p>
+            <p>
+                Posted {postedTime} on {postedDate}
+            </p>
+            <img src={article.article_img_url} />
```

Comment:
Good job showing the data - but can we do it in a more interesting way that doesn't rely on just labels? Check out [this article](https://www.refactoringui.com/previews/labels-are-a-last-resort)